### ISSUE TITLE
Grade

### ISSUE THREAD

**Issue by ericayin** [MichaelRoe1/RubyOnRails#1]
0/2

No changes in the commit including routes.rb
The structuring of your repo is kind of strange, if you did do it but the files are private or something, talk to Howard about it

**Comment by MichaelRoe1** [comment]
Hey Eric, I'm definitely having problems structuring my Github repo then, because I did the assignment in full. I'm going to talk to Howard after class this week and settle the issue. 

Given that on my local machine, the homework is completed (with a timestamp) and I can render my homepage, could I possibly get credit retroactively? 

I tried really hard to follow the instructions here for submission: https://help.github.com/articles/adding-an-existing-project-to-github-using-the-command-line/ for me it would be a shame not to receive any credit : ((

Let me know if you have any thoughts about this. Thanks!
